src/main/java/frc/robot/systems/DriveFSMSystem.java

```diff
@@ -256,7 +308,9 @@ private void handleForwardOrBackwardState(TeleopInput input,
 		System.out.println("Inches: " + inches);
 		System.out.println("Error: " + error);
 
-		// Try This Next Time:
+		// Checks if either the encoder value is equal to the inches required (reached destination)
+		// or checks if the robot has hit a wall (encoder value is not chaning but motor power is
+	// not zero)
 		if ((inches > 0 && error < Constants.ERR_THRESHOLD_STRAIGHT_IN)
 			|| (inches < 0 && error > -Constants.ERR_THRESHOLD_STRAIGHT_IN)) {
 			System.out.println("im here");
```

Comment:
debug print statements shouldn't be included in PR, or at least make them more descriptive

Reply:
Yea we'll remove them